app/src/main/AndroidManifest.xml

```diff
@@ -10,16 +10,18 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingApp"
-        tools:targetApi="31">
+        tools:targetApi="31" >
```

Comment:
Do you need to specify the targetApi here? The best way to set the targetApi depends on your specific needs. In general, it is best to set the targetApi in your app's build.gradle file. This will ensure that all activities in your app are compiled and run on the same API level.

Reply:
I don't really know what happened here. I'm aware that the sdk should be defined in the build file
I was working with the navigation, and I was messing around today with the activities provided by the IDE.
I guess it messed up the manifest and I didn't see it
I'll remove it

Reply:
I just checked the file history, that attribute is there since the beginning.
Even if I remove it, the IDE tells me that it should be there

<img width="456" alt="image" src="https://github.com/CFedericoDavid/g-reskilling-android-parkingApp/assets/13878290/d24fdb45-6268-4df8-8ef6-0dddc201a194">